src/BenchmarksDriver2/benchmarks.proxy.yml

```diff
@@ -12,6 +12,16 @@ variables:
     path: /
 
 jobs:
+  yarp:
+    source:
+      repository: https://github.com/microsoft/reverse-proxy.git
+      branchOrCommit: halter73/benchmark
```

Comment:
This should be `master` once https://github.com/microsoft/reverse-proxy/pull/148 is merged.

Reply:
we can change it once it's merged, and point to this branch for now.

Reply:
Why not have a .yml file describing the app alongside the benchmarks app, and `import` it?